src/js/components/TextInput.js

```diff
@@ -236,6 +237,10 @@ export default class TextInput extends Component {
           this._onAddDrop(event);
         }
       }
+
+      if(event.keyCode === tab) {
+        this.setState( { focused: false });
```

Comment:
totally minor, would you mind removing the extra space before the curly bracket ? We should add that to our linter :)

`this.setState({ focused: false });`